flepimop/gempyor_pkg/src/gempyor/file_paths.py

```diff
@@ -58,13 +59,7 @@ def create_file_name(
     """
     if create_directory:
         os.makedirs(
-            create_dir_name(
-                run_id,
-                prefix,
-                ftype,
-                inference_filepath_suffix,
-                inference_filename_prefix,
-            ),
+            create_dir_name(run_id, prefix, ftype, inference_filepath_suffix, inference_filename_prefix,),
```

Comment:
Line length of 106.

Reply:
I remember the line limit is 120, because the gitbook black example is:  black --line-length 120 xxx